.github/workflows/ci.yml

```
@@ -12,297 +10,319 @@
+      id: cache-wails
+      uses: actions/cache@v4
+      with:
+        path: ~/go/bin/wails
+        key: ${{ runner.os }}-wails-v2.10.1
+    
+    # Install Wails (all platforms)
+    - name: Install Wails
+      if: steps.cache-wails.outputs.cache-hit != 'true'
+      shell: bash
       run: go install github.com/wailsapp/wails/v2/cmd/wails@v2.10.1
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)